packages/webvh/src/dids/didWebvhUtil.ts

```diff
@@ -0,0 +1,94 @@
+import { type AgentContext, Buffer, DidDocument, JsonTransformer, KeyType, Key } from '@credo-ts/core'
+import { createDID, AbstractCrypto, type SigningOutput, type SigningInput } from 'didwebvh-ts'
+
+export class DIDWebvhCrypto extends AbstractCrypto {
+  private agentContext?: AgentContext
+
+  public constructor(agentContext?: AgentContext) {
+    super({
+      verificationMethod: {
+        id: 'did:webvh:123',
+        controller: 'did:webvh:123',
+        type: 'Ed25519VerificationKey2020',
+        publicKeyMultibase: '123',
+        secretKeyMultibase: '123',
+      },
+    })
+    this.agentContext = agentContext
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public async sign(input: SigningInput): Promise<SigningOutput> {
+    throw new Error('Not implemented')
+  }
+
+  public async verify(signature: Uint8Array, message: Uint8Array, publicKey: Uint8Array): Promise<boolean> {
```

Comment:
It's probably good if the interface from did web vh passes the keytype as well, so it's ready to support additional key types as well